pf/tfbridge/provider_create.go

```diff
@@ -44,6 +45,23 @@ func (p *provider) CreateWithContext(
 
 	priorState := newResourceState(ctx, &rh)
 
+	for key, fld := range rh.pulumiResourceInfo.Fields {
+		if _, ok := checkedInputs[resource.PropertyKey(key)]; !ok && fld.Default != nil {
+			// using default value for empty property
+			v, err := convert.GetDefaultValue(fld.Default, &tfbridge.PulumiResource{
+				URN:        urn,
+				Properties: checkedInputs,
+				Seed:       make([]byte, 0),
```

Comment:
This `Seed` bit is probably not right. 

I've looked closely at pkg/tfbridge/provider.go to see how we currently do this in prod.

It looks like this needs to move into `CheckWithContext`. Pulumi calls Check on a resource to find out "final inputs" and it'a good place to apply Pulumi-level defaults; also what pre-PF implementation does. Then Pulumi will pass the result of Check to the Create implementation. So Create does not need to do it again.

You will notice that `randomSeed []byte` parameter is expoed to CheckWithContext, this is what's needed to be passed here.

Reply:
Moved the code to `CheckWithContext`